5_analyze_data/notebooks/KS-test_analysis/kstest_utils.py

```diff
@@ -0,0 +1,75 @@
+from scipy.stats import ks_2samp
+import pathlib
+import pandas as pd
+
+def nf1_ks_test_two_sample(normalized_data: pd.DataFrame):
```

Comment:
```suggestion
def nf1_ks_test_two_sample(normalized_data: pd.DataFrame) -> pd.DataFrame:
```

Reply:
Added in [f66e49a](https://github.com/WayScience/NF1_SchwannCell_data/pull/42/commits/f66e49a88b1ca6f0b20d5be66096be358ed24d8f)